src/applications/apps.ts

```diff
@@ -189,7 +211,7 @@ export function unloadApplication(appName, opts = { waitForUnmount: false }) {
       )
     );
   }
-  const app = find(apps, (App) => toName(App) === appName);
+  const app = apps.find((app) => toName(app) === appName);
```

Comment:
This is a runtime change switching to Array.prototype.find rather than the IE-workaround implementation. See https://single-spa.js.org/blog/2023/11/27/single-spa-6#browser-support--ie11